spec/requests/todos_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Todos', type: :request do
+  describe 'GET /todos' do
+    subject(:get_todos) { get '/todos' }
+
+    context 'with no todos added' do
+      let(:empty_array) { [] }
+
+      before { get_todos }
```

Comment:
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Reply:
c92c3ccb8168843d88483ffa75f81eef62f71b66